.github/workflows/CI.yml

```diff
@@ -47,9 +40,28 @@ jobs:
         distribution: temurin
     - name: Build and Run Tests
       run: |
-          ./gradlew check
+          ./gradlew check -x integTest -x yamlRestTest
```

Comment:
Based on our discussion on #84, would code coverage just run for unit tests now?